pkg/csi/service/common/authmanager.go

```diff
@@ -505,10 +505,10 @@ func getFSEnabledClustersWithPriv(ctx context.Context, vc *cnsvsphere.VirtualCen
 			return nil, err
 		}
 		if !(*config.Enabled) {
-			log.Debugf("cluster: %+v is a non-vSAN cluster. Skipping this cluster", cluster)
+			log.Infof("cluster: %+v is a non-vSAN cluster. Skipping this cluster", cluster)
 			continue
 		} else if config.FileServiceConfig == nil {
-			log.Debugf("VsanClusterGetConfig.FileServiceConfig is empty. Skipping this cluster: %+v with config: %+v",
+			log.Infof("VsanClusterGetConfig.FileServiceConfig is empty. Skipping this cluster: %+v with config: %+v",
```

Comment:
printing config of the cluster is not required with info level logging.

Reply:
fixed.